src/result_gen_with_api.py

```diff
@@ -322,6 +327,7 @@ def main():
     depthmap_img_workflow_path = args.depthmap_img_workflow_path
     height_workflow_artifact_path = args.height_workflow_artifact_path
     height_workflow_scan_path = args.height_workflow_scan_path
```

Comment:
Now that we have different types of models, we should name this model more specifically.
e.g. `height_workflow_scan_path` -> `height_plaincnn_workflow_scan_path `

There are many more places in this code which would then adjust older code, so this could be done in a separate PR